src/main/java/fr/insee/genesis/domain/ports/api/SurveyUnitApiPort.java

```diff
@@ -22,6 +23,7 @@ public interface SurveyUnitApiPort {
     Stream<SurveyUnitModel> findByIdQuestionnaire(String idQuestionnaire);
 
     List<SurveyUnitModel> findLatestByIdAndByIdQuestionnaire(String idUE, String idQuest);
+    SurveyUnitPerret findLatestByIdAndByIdQuestionnairePerret(String idUE, String idQuest);
```

Comment:
findLatestByStateByidAndByIdQuestionnaire
I think that we don't have to mention Perret in our code